model_pdraw.go

```diff
@@ -227,8 +248,8 @@ func pdrawAuthorized(gid GoogleID, oid OperationID) (bool, TeamID, error) {
 
 // insertMarkers adds a marker to the database
 func (o *Operation) insertMarker(m Marker) error {
-	_, err := db.Exec("INSERT INTO marker (ID, opID, PortalID, type, gid, comment, complete) VALUES (?, ?, ?, ?, ?, ?, ?)",
-		m.ID, o.ID, m.PortalID, m.Type, m.AssignedTo, m.Comment, m.Complete)
+	_, err := db.Exec("INSERT INTO marker (ID, opID, PortalID, type, gid, comment, state) VALUES (?, ?, ?, ?, ?, ?, ?)",
```

Comment:
Does db.Exec verify that no SQL inyection can happen here?

Reply:
No code injection can take place. The way I build the statements is safe, verified by testing and gosec static validation.